### ISSUE TITLE
FeatureTable widget throws 'module not found' error

### ISSUE THREAD

**Issue by ReneU** [Esri/arcgis-js-cli#71]
## Description

The build crashes with errors if the FeatureTable widget is imported. It seems like webpack is not able to resolve the FeatureTable's dependencies. 

## Actual Behavior

![image](https://user-images.githubusercontent.com/5748802/79437477-86398e00-7fd2-11ea-8328-77ced66f4e55.png)

## Steps to Reproduce

1. Create app with cli.
2. Import `esri/widgets/FeatureTable` in any component.
3. run `npm start`

## Your Environment

* Version used: 4.15 (with react template)
* Browser Name and version:
* Operating System and version (desktop or mobile):
* Link to your project if available:

Thanks!

**Comment by odoe** [comment]
I'll get this updated in the template apps, but the fix you can apply yourself is to update the `webpack.config.node` section.

```js
  node: {
    process: false,
    global: false,
    setImmediate: false, // add this!
    fs: "empty"
  }
```

**Comment by ReneU** [comment]
This works, thanks!

**Comment by odoe** [comment]
This should be fixed in [4.15.1](https://github.com/Esri/arcgis-js-cli/releases/tag/4.15.1)
